frontend/customization/config/global.json

```diff
@@ -10,10 +10,8 @@
   "apiUrl": "https://geotrekdemo.ecrins-parcnational.fr/api/v2",
   "reservationPartner": "parcdesecrins",
   "reservationProject": "parcdesecrins",
-  "googleAnalyticsId": "G-8FSV2N4FXN",
-  "googleSiteVerificationToken": "eKAyxwaXAobFWQcJen0mnZ8T3CpLoN45JysXeNkRf38",
   "baseUrl": "http://localhost:3000",
-  "fallbackImageUri": "https://upload.wikimedia.org/wikipedia/fr/d/df/Logo_ecrins.png",
+  "fallbackImageUri": "https://demo-rando.geotrek.fr/images/placeholder.png",
   "touristicContentLabelImageUri": "https://upload.wikimedia.org/wikipedia/fr/d/df/Logo_ecrins.png",
```

Comment:
Maybe `touristicContentLabelImageUri` should have the same value of `fallbackImageUri`